### DISCUSSION TITLE
Plot individual agent attribute with datacollector.get_agent_vars_dataframe()

### DISCUSSION THREAD

**Discussion by erissatallan** [mesa/mesa#1556] [Q&A] [answered]
I'm trying to figure out a way to plot an agent attribute I've called `assurance_probability` by perhaps specifying their ID while plotting `datacollector.get_agent_vars_dataframe()`.

At present my code look like this
```
if __name__ == "__main__":
    economyModel = EconomyModel(4, 10, 10, liquidityPoolModel)
    economyModel.execute_model(30)
    agent_assurance_probabilities = economyModel.datacollector.get_agent_vars_dataframe()
    print(agent_assurance_probabilities.head(n=40))
```
with sample output
![agent_assurance_probabilities](https://user-images.githubusercontent.com/92621678/206619183-20689a91-66a6-4e80-ab28-49756e470c79.png)

**Comment by rht** [comment, marked as answer]
`agent_assurance_probabilities` is a `DataFrame`, and so you can do the usual DF filtering operation: `agent_assurance_probabilities[agent_assurance_probabilities.AgentID == x]`, and plot the timeseries.
